src/doc/ESMF_quickstart.tex

```diff
@@ -174,19 +174,19 @@ \subsubsection{Environment variables}
 
 \subsubsection{GNU make}
 The ESMF build system uses the GNU make program; it is normally named 
-{\tt gmake} but may also be simply {\tt make} or {\tt gnumake} on some 
-platforms (we will use {\tt gmake} in this document). ESMF does not use
+{\tt make} but may also be {\tt gmake} or {\tt gnumake} on some 
+platforms. We will use {\tt make} in this document. ESMF does not use
```

Comment:
After "We will use {\tt make} in this document", I'm wondering if it would be worth explicitly adding something like, "but note that you may need to replace this usage with {\tt gmake} or {\tt gnumake} to ensure the use of the appropriate make program on your system". I know this kind of repeats what is already said, but it feels like this is something that could trip people up (it has definitely tripped me up in other contexts in the past), so I wonder if it's worth being really explicit here.

Reply:
I agree that this could be more explicit. I just pushed a change to better clarify this in `user_make.tex`, which now is used where the GNU Make dependency is discussed. Please review once the online branch docs have rebuilt. Thanks!